packages/app/setup.py

```diff
@@ -63,6 +63,22 @@ def get_var(var_name):
     'galaxy.tools.toolbox.lineages',
     'galaxy.tools.util',
     'galaxy.tools.util.galaxyops',
+    'galaxy.tool_shed',
+    # TODO:
+    # 'galaxy.tool_shed.galaxy_install',
```

Comment:
Can you resolve these todos to complete the transition? If I understand this correctly`galaxy_install` is used only by Galaxy but is located in 'lib/tool_shed' and imports toolshed utilities which have not been moved to 'lib/galaxy/tool_shed' yet.

Reply:
I'm happy to complete the transition - I was hoping to get this piece merged to ensure the conflict with #8830 wouldn't prevent this from being merged and the approach would not be -1'd by anyone before continuing but I am committed to getting this done. A smaller PR is inline with say https://github.com/galaxyproject/galaxy/pull/8280 - not resulting in the perfect package structure but an incremental step in the correct direction.

If I have your assurance this approach is acceptable though I will continue to work on this. If your okay with the approach is there a reason to not merge this piece as is?

Reply:
That is all of galaxy_install that is needed by Galaxy after the controller changes in your PR I suppose. No need to transfer grids or utility containers I suppose?

Reply:
Right, and I think it would be cool to complete this transition within this PR i.e. move `galaxy_install` to `lib/galaxy/tool_shed`. #8830 can wait for that.